src/checks.rs

```diff
@@ -1466,8 +1466,8 @@ impl CheckKind {
             CheckKind::SuperCallWithParameters => {
                 "Use `super()` instead of `super(__class__, self)`".to_string()
             }
-            CheckKind::UnnecessaryFutureImport(name) => {
-                format!("Unnessary __future__ import `{name}` for target Python version")
+            CheckKind::UnnecessaryFutureImports(name) => {
```

Comment:
Nit: can we make this `names` and take a vector, rather than a pre-formatted string? Then we can split the check to have proper pluralization, like:

```rust
CheckKind::DuplicateHandlerException(names) => {
    if names.len() == 1 {
        let name = &names[0];
        format!("Exception handler with duplicate exception: `{name}`")
    } else {
        let names = names.iter().map(|name| format!("`{name}`")).join(", ");
        format!("Exception handler with duplicate exceptions: {names}")
    }
}
```

(Using a vector as the argument, rather than a pre-formatted string, will also yield a nicer fixture file, since the list of names will be structured data rather than a comma-joined string.)